spec/features/access_points/collection_spec.rb

```diff
@@ -9,13 +9,13 @@
 
   scenario "Access point masthead should be visible with 1 course reserve document" do
     expect(page).to have_title("Image Collection1 Collection in SearchWorks catalog")
```

Comment:
Can we move this whole file out of `/access_points/` since there no longer is a collection access point?